src/ppx/application.cpp

```diff
@@ -1007,10 +1008,16 @@ int Application::Run(int argc, char** argv)
     }
 #endif
 
-    mMaxFrame = UINT64_MAX;
-    // If command line provided a maximum number of frames to draw
-    if (mStandardOptions.frame_count != -1) {
-        mMaxFrame = mStandardOptions.frame_count;
+    mMaxFrames = UINT64_MAX;
+    // If command line provided a maximum number of frames to draw.
+    if (mStandardOptions.frame_count > 0) {
```

Comment:
If you change frame_count behavior in that PR too you should update the title to reflect this change.